content/posts/2023_01_29_ifk_smhi.md

```diff
@@ -0,0 +1,49 @@
+title: Python-klient för smhi-data
+date: 2023-01-29
+modified:
+author: Anders Nord
+category: Projects
+tags: python, smhi, weather, data, climate, ifk
+data:
+status: published
+
+<div>
```

Comment:
You don't need to write html just write markdown as you did before, for text and images. Sorry if my example made it unclear... I was mainly point to the header and file structure